config/webpacker.yml

```
@@ -51,10 +51,9 @@ development:
     watch_options:
       ignored: /node_modules\/(?!@18f\/identity-)/
 
-
 test:
   <<: *default
-  compile: true
+  compile: false
```

Is this an ERB'd file? If so, we could do something like...

```suggestion
  compile: <%= !ENV['CI'] %>
```

---

> Is this an ERB'd file? If so, we could do something like...

That would make things a lot easier, but I don't think so unfortunately. At least from what I can tell, it's loaded as plain YAML.

https://github.com/rails/webpacker/blob/beb645ecbf707eb4f3311e61b05abb67fe0ca010/lib/webpacker/configuration.rb#L76-L81

---

🤠 😭  I think ERB'ing YAML is an anti-pattern....but the one time it would have helped us.... it's not there